Diagnostics/HealthChecker/DataCollection/ExchangeInformation/Get-ExchangeInformation.ps1

```diff
@@ -372,6 +372,13 @@
 
         if ($buildInformation.ServerRole -ne [HealthChecker.ExchangeServerRole]::Edge) {
             $exchangeInformation.ApplicationPools = Get-ExchangeAppPoolsInformation
+            try {
+                $hybridConfiguration = Get-HybridConfiguration -ErrorAction Stop
```

Comment:
Let's just do the following, as there isn't a reason to store the results in a variable. 

```
$exchangeInformation.GetHybridConfiguration = Get-HybridConfiguration -ErrorAction Stop
```